openapi3.yaml

```diff
@@ -7,6 +7,31 @@ info:
     name: MIT
     url: https://opensource.org/licenses/MIT
 paths:
+  /storage:
+    get:
+      tags:
+        - storage
+      summary: Get free, in-use and total storage size for exporting (in bytes)
+      operationId: getStorage
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/StorageResponse'
+        '404':
```

Comment:
why 404?

Reply:
In case a bad path is given as gpkg location.